data-prepper-plugins/event-json-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/event_json/EventJsonOutputCodec.java

```diff
@@ -44,22 +44,24 @@ public String getExtension() {
     public void start(OutputStream outputStream, Event event, OutputCodecContext context) throws IOException {
         Objects.requireNonNull(outputStream);
         generator = factory.createGenerator(outputStream, JsonEncoding.UTF8);
-        generator.writeStartObject();
+        generator.writeStartArray();
```

Comment:
We should keep the `writeStartObject`.

This could allow us to also include metadata such as the version.

e.g.

```
{
  "metadata" : {"version" : "2.8"},
  "events" : [
    {"data":...,"metadata"},
    {"data":...,"metadata"},
    {"data":...,"metadata"},
  ]
}
```

See this code:

https://github.com/opensearch-project/data-prepper/blob/f9a3a60975f65d58261d1a37e3b793796d52bc8a/data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/codec/json/JsonOutputCodec.java#L55-L57

Though, I don't think we need to make the `events` key configurable.

Reply:
Good point. But the following should be enough.
```
{
  "version" : "2.8",
  "events" : [
    {"data":...,"metadata"},
    {"data":...,"metadata"},
    {"data":...,"metadata"},
  ]
}
```